e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/single-contract-policy/save-and-back.spec.js

```diff
@@ -0,0 +1,171 @@
+import {
+  add,
+  getDate,
+  getMonth,
+  getYear,
+  sub,
+} from 'date-fns';
+import { submitButton, saveAndBackButton } from '../../../../pages/shared';
+import { typeOfPolicyPage, singleContractPolicyPage } from '../../../../pages/insurance/policy-and-export';
+import partials from '../../../../partials';
+import { TASKS } from '../../../../../../content-strings';
+import { ROUTES, FIELD_IDS } from '../../../../../../constants';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const { taskList } = partials.insurancePartials;
+
+const singlePolicyFieldId = FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS.POLICY_TYPE;
+const singlePolicyField = typeOfPolicyPage[singlePolicyFieldId].single;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        REQUESTED_START_DATE,
+      },
+    },
+  },
+} = FIELD_IDS;
+
+context('Insurance - Policy and exports - Type of policy page - Save and go back', () => {
+  let referenceNumber;
+  const date = new Date();
+  const futureDate = add(date, { days: 1, months: 3 });
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
+
+    singlePolicyField.input().click();
+    submitButton().click();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.SINGLE_CONTRACT_POLICY}`;
```

Comment:
can we destructure please? this one is big :P

Reply:
Good shout